src/NUnitFramework/Directory.Build.props

```diff
@@ -1,7 +1,7 @@
 ﻿<Project>
 
   <PropertyGroup>
-    <LangVersion Condition="'$(MSBuildProjectExtension)' == '.csproj'">13</LangVersion>
+    <LangVersion Condition="'$(MSBuildProjectExtension)' == '.csproj'">14</LangVersion>
```

Comment:
Good catch. It looks like we missed this earlier.
Should we continue to target a specific version here, or is `latest` sufficient?